### DISCUSSION TITLE
Resize two windows at once when windows are adjecent

### DISCUSSION THREAD

**Discussion by kan-ish** [rxhanson/Rectangle#1324] [Ideas]
Sometimes windows are right next to each other and you want to make one bigger but want the other window to shrink an equal amount as the other window gets bigger. Right now you'll go about doing that by making one bigger by grabbing it from the side, and then similarly making the other one smaller. There could be a feature in Rectangle that grabs the edge of bth the windows at once and resizes them as we move the cursor. Exactly like MS Windows' default window manager does it.

https://github.com/rxhanson/Rectangle/assets/67118535/a239ea94-e8be-4a61-a620-14104bb750d4

**Comment by kan-ish** [comment]
Or an option to add custom window size/saving a window configuration could partially solve this problem

**Comment by Aaron-Rumpler** [comment]
Both features are included in [Rectangle Pro](https://rectangleapp.com/pro).

**Comment by rxhanson** [comment]
Indeed, they are both in Rectangle Pro (thanks, @Aaron-Rumpler!). 

Here some corresponding discussions on how to use them:

Adjacent window resizing:
https://github.com/rxhanson/RectanglePro-Community/discussions/429

Window layouts:
https://github.com/rxhanson/RectanglePro-Community/discussions/111

(Rectangle has a smaller feature set than Pro to keep the app more approachable to most users.)

**Reply by kan-ish** [reply]
Alright thanks

**Comment by joshstobbs** [comment]
Piggybacking on this discussion, is there a way to resize left and right with my keyboard?  I basically am looking to emulate dragging the side of the app but by pressing a hotkey instead.

**Reply by thenickygee** [reply]
ps. @joshstobbs, @rxhanson answered this in #1599
